src/Message/AuthorizeRequest.php

```diff
@@ -176,6 +194,32 @@ public function setApplicationFee($value)
         return $this->setParameter('applicationFee', $value);
     }
 
+    /**
+     * @return float
+     */
+    public function getApplicationFeeAmount()
+    {
+        return $this->getParameter('applicationFeeAmount');
+    }
+
+    /**
+     * @return int
+     */
+    public function getApplicationFeeAmountInteger()
+    {
+        return (int) round($this->getApplicationFeeAmount() * pow(10, $this->getCurrencyDecimalPlaces()));
+    }
+
+    /**
+     * @param string $value
+     *
+     * @return AbstractRequest provides a fluent interface.
+     */
+    public function setApplicationFeeAmount($value)
+    {
+        return $this->setParameter('applicationFeeAmount', $value);
```

Comment:
Ditto. Keep this the same as before and make the driver translate to this.

Reply:
done